contrib/opentelemetry/go.mod

```diff
@@ -4,14 +4,14 @@ go 1.20
 
 require (
 	github.com/stretchr/testify v1.9.0
-	go.opentelemetry.io/otel v1.21.0
-	go.opentelemetry.io/otel/sdk v1.21.0
-	go.opentelemetry.io/otel/trace v1.21.0
+	go.opentelemetry.io/otel v1.23.0
+	go.opentelemetry.io/otel/sdk v1.23.0
+	go.opentelemetry.io/otel/trace v1.23.0
```

Comment:
I assume 1.23 instead of 1.26 just because we want lowest-working version?

Reply:
Yeah, this was the lowest version that our old implementation broke on